platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -343,7 +332,17 @@ void updateSepTransaction(PlatformTransactionData patch, JdbcSepTransaction txn)
           txnUpdated = updateField(patch, sep24Txn, "memoType", txnUpdated);
         }
 
+        // add a memo for the transaction if the transaction is ready for user to send funds
+        if (sep24Txn.getMemo() == null
+            && Kind.WITHDRAWAL.getKind().equals(sep24Txn.getKind())
+            && sep24Txn.getStatus().equals(PENDING_USR_TRANSFER_START.toString())) {
+          SepDepositInfo sep24DepositInfo = sep24DepositInfoGenerator.generate(sep24Txn);
+          sep24Txn.setMemo(sep24DepositInfo.getMemo());
```

Comment:
I think we need to keep setting `withdraw_anchor_account` otherwise this is a breaking change.

Reply:
Actually both `to_account` and `withdraw_anchor_account` were already set to asset's distribution account when the transaction is created ([here](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java#L216))

I don't think it make any sense to reset them to the same value here...

Reply:
Didn't see that, thanks!